src/assets/styles/collections/_projects.css

```diff
@@ -153,7 +153,6 @@
         grid-column: 2;
         height: 100%;
         padding-block-start: 5.5rem;
```

Comment:
I think this will work better with flex display and the content vertically centered.